packages/notification/docs/InlineNotification.stories.tsx

```diff
@@ -49,245 +20,134 @@ export default {
-
-export const DismissiblePositiveAutohideHideCloseIcon = () => (
-  <InlineNotification
-    type="positive"
-    title="Success"
-    autohide
-    hideCloseIcon
-    automationId="notification1"
-  >
-    New user data, imported by mackenzie@hooli.com has successfully uploaded.{" "}
-    <a href="/">Manage users is now available</a>
-  </InlineNotification>
-)
-DismissiblePositiveAutohideHideCloseIcon.storyName =
-  "Dismissible, Positive, Autohide, Hide Close Icon"
-
-export const DismissibleInformative = () => (
-  <InlineNotification
-    type="informative"
-    title="Informative"
-    automationId="notification1"
-  >
-    New user data is currently being processed. We'll let you know when the
-    process is completed. <a href="/">Manage users</a>
-  </InlineNotification>
-)
-DismissibleInformative.storyName = "Dismissible, Informative"
-
-export const DismissibleCautionary = () => (
-  <InlineNotification
-    type="cautionary"
-    title="Warning"
-    automationId="notification1"
-  >
-    New user data, imported by mackenzie@hooli.com has uploaded with some minor
-    issues. <a href="/">View issues</a>
-  </InlineNotification>
-)
-DismissibleCautionary.storyName = "Dismissible, Cautionary"
-
-export const DismissibleNegative = () => (
-  <InlineNotification
-    type="negative"
-    title="No network connection"
-    automationId="notification1"
-  >
-    Check your connection and try again. <a href="/">Refresh</a>.
-  </InlineNotification>
-)
-DismissibleNegative.storyName = "Dismissible, Negative"
+DefaultKaizenDemo.storyName = "Default (Kaizen Demo)"
+
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => {
+  const forceMultilineText = (
+    <>
+      This is a short line which we want to see underneath the notification
+      heading.
+      <ol>
+        <li>Contact support</li>
+        <li>Contact your admin</li>
+      </ol>
+    </>
+  )
+  const multilineText = (
+    <>
+      There’s a problem connecting to your HRIS. Check your HRIS is working and
+      check your <a href="/">integration settings</a>, or if you require more
+      assistance please <a href="/">contact support</a>... or just don't do
+      anything and observe that this notification contains an absurd amount of
+      text that is purposely verbose in order to demonstrate that verbosity is,
+      in most cases, just in general really, i guess it's debatable, unnecessary
+      and to demonstrate that verbosity makes this notification's body text spit
+      into multiple lines because there is, surely, unequivocally, no way that
+      all of this can fit into one line of text on an average screen...
+    </>
+  )
+  return (
+    <>
+      <StoryWrapper isReversed={isReversed}>
+        <Heading
+          variant="heading-3"
+          tag="h1"
+          color={isReversed ? "white" : "dark"}
+        >
+          Prominent
+        </Heading>
+        <StoryWrapper.RowHeader headings={["Dismissible"]} />
```

Comment:
```suggestion
```

Since `Persistent` is now a row, this doesn't make sense to be the column heading.